<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CrewManager</class>
 <widget class="QWidget" name="CrewManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>896</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>861</width>
     <height>421</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(59, 59, 88);
border-radius:10px;
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>25</y>
      <width>341</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>27</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: qlineargradient(spread:pad, x1:0, y1:0.460227, x2:1, y2:0.46, stop:0 rgba(231, 126, 186, 255), stop:1 rgba(170, 85, 255, 255));</string>
    </property>
    <property name="text">
     <string>Crew-Management</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="exit">
    <property name="geometry">
     <rect>
      <x>835</x>
      <y>5</y>
      <width>22</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#exit{
background-color: qlineargradient(spread:pad, x1:0, y1:0.460227, x2:1, y2:0.46, stop:0 rgba(231, 126, 186, 255), stop:1 rgba(170, 85, 255, 255));
border-radius:10px;
font: 10pt &quot;Segoe UI&quot;;
color:rgb(200, 200, 200);
}
QPushButton#exit:hover {
	font:  75 12pt &quot;Segoe UI&quot;;
	color: rgb(255, 255, 255);
}

QPushButton#exit:pressed{
	
	color: rgb(59, 59, 88);
}</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>771</width>
      <height>311</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(39, 39, 60);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QListWidget" name="crewList">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>11</y>
       <width>741</width>
       <height>291</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 20pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>30</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#backButton{
color: qlineargradient(spread:pad, x1:0, y1:0.460227, x2:1, y2:0.46, stop:0 rgba(231, 126, 186, 255), stop:1 rgba(170, 85, 255, 255));
font: 30pt &quot;Segoe UI&quot;;

}
QPushButton#backButton:hover {
	font:  75 25pt &quot;Segoe UI&quot;;
}

QPushButton#backButton:pressed{
	color: rgb(59, 59, 88);
}</string>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addPersonButton">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>49</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>25</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#addPersonButton{
background-color: qlineargradient(spread:reflect, x1:0, y1:1, x2:1, y2:0.307, stop:0.0909091 rgba(244, 108, 203, 255), stop:1 rgba(247, 127, 127, 255));
border-radius:15px;
font: 25pt &quot;Segoe UI&quot;;
color:rgb(200, 200, 200);

}
QPushButton#addPersonButton:hover {
	font:  75 22pt &quot;Segoe UI&quot;;
	color: rgb(255, 255, 255);
}

QPushButton#addPersonButton:pressed{
	
	color: rgb(59, 59, 88);
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
